commands/root.go

```diff
@@ -23,14 +23,15 @@ import (
 	"os/signal"
 	"strings"
 
-	"github.com/c-bata/go-prompt"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
+	"github.com/hazelcast/hazelcast-commandline-client/go-prompt"
+
 	clusterCmd "github.com/hazelcast/hazelcast-commandline-client/commands/cluster"
+	sqlCmd "github.com/hazelcast/hazelcast-commandline-client/commands/sql"
```

Comment:
The import aliases should be all-lowercase, like `sqlcmd`.

Reply:
Thanks, fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/58/commits/bb113e0a0f4a4c4dea35ae9618a70731aaf8d852